tests/test_client.py

```
@@ -1,12 +1,16 @@
 """Tests for Zammad client configuration and error handling."""
 
+import logging
 import os
 from unittest.mock import MagicMock, patch
 
 import pytest
 
 from mcp_zammad.client import ConfigException, ZammadClient
 
+# Non-secret placeholder for tests (avoids bandit S106 on http_token kwargs)
+_TEST_HTTP_TOKEN = "test-http-token"
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to: "_TEST_HTTP_TOKEN"

(S105)